intro.tex

```diff
@@ -26,45 +26,47 @@ \section{Introduction And Related Work}
 publicly available platforms that can be used in fog contexts --- are rare.
 
 In this paper, we introduce our Seattle platform and make the case for
-why its unique designed is well suited for \gls{fc} applications. But,
+why its unique design is well suited for \gls{fc} applications. But,
 to provide context, we first review relevant aspects of the existing
 literature.
 
 \subsection{Related Work}
 
-Emerging technologies are a prospective area for research in \gls{fc}.
-Bellavista et al.~\cite{bellavista_feasibility_2017} investigate the
+Emerging technologies have opened potentially promising new areas for research
+in \gls{fc}. Bellavista et al.~\cite{bellavista_feasibility_2017} investigate the
 applicability of Docker containers for \gls{fc}. The work uses and extends
 the open-source Kura framework to create \gls{IoT} gateways that control information
 flow between fog nodes and the cloud, while also reviewing and benchmarking
-different container related technologies used in the fog nodes.
-The focus of the work tends towards \gls{IoT} computing, where tailored services run
+different container-related technologies used in these nodes.
+The focus of this work is on \gls{IoT} computing, where tailored services run
 on fog nodes to gather data that is forwarded to the cloud for further processing.
 A similar evaluation of Docker containers for edge computing can be found in
 \cite{ismail_evaluation_2015}.
 
 Others have contributed fog platforms for very specific use cases, e.g.
 an emergency alert system using smartphones that propagate alerts to nearby
-emergency departments \cite{7134091}; an idea also proposed by Masip
+emergency departments \cite{7134091}, an idea also proposed by Masip-Bruin
 et al.~\cite{masip-bruin_foggy_2016}. Gazis et al. introduce an ``Adaptive
 Operations Platform''
-to effectively apply equipment failure models within the context of Industrial
+to effectively apply equipment failure models to the fog computing
+infrastructure of Industrial
```

Comment:
I don't understand what the "fog computing infrastructure of Industrial IoT" means, so I didn't adopt that edit.

Reply:
On a few of these related work summaries, I went to the source material to try to make the statements about it a bit more specific and less generic. I admit that this one was a bit clumsy. Its probably fine as is and probably not worth the effort of going back to the paper and trying to extract a more meaningful statement.